src/NuGet.Indexing/StorageLoader.cs

```diff
@@ -44,6 +41,35 @@ public JsonReader GetReader(string name)
             }
         }
 
+        public DateTime? GetLastUpdateTime(string name)
+        {
+            try
+            {
+                _logger.LogInformation("StorageLoader.GetLastUpdateTime: {ReaderTarget}", name);
+
+                var blob = GetBlob(name);
+                if (!blob.Exists())
+                {
+                    return null;
+                }
+
+                blob.FetchAttributes();
+
+                // LastModified time is always in UTC for Azure blobs
+                var lastModifiedUtcTime = blob.Properties.LastModified.HasValue
+                    ? blob.Properties.LastModified.Value
+                    : DateTimeOffset.MinValue;
```

Comment:
Please take a look at the latest commit. https://github.com/NuGet/NuGet.Services.Metadata/pull/128/commits/baa858d42373598638af4d0061054c46be3a515b#diff-abd3dac6b87987c81b2d39954087afdaR64